Formula/hyperkit.rb

```diff
@@ -0,0 +1,56 @@
+class Hyperkit < Formula
+  desc "Toolkit for embedding hypervisor capabilities in your application"
+  homepage "https://github.com/moby/hyperkit"
+  url "https://github.com/moby/hyperkit.git",
+    :tag => "v0.20180123",
+    :revision => "8bbe1da0166553a5b0ccff170bf9ccbd7d10753e"
+
+  head do
+    url "https://github.com/moby/hyperkit.git"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+  end
+
+  option "with-qcow", "Enable support for qcow disk image format"
```

Comment:
Remove option, build by default

Reply:
We want to ship as many formulas as possible without options, providing a full feature set for users without the need to compile themselves.